scripts.d/.env-cfd-dev

```diff
@@ -15,8 +15,8 @@ PYTHON_VERSION=$(python3 --version | cut -d' ' -f2)
 # And remove the micro version
 PYTHON_VERSION=${PYTHON_VERSION%.*}
 
-[[ -f ${SCBI_ENV_DIR}/.env-ng-shared-defs ]] &&
-    source ${SCBI_ENV_DIR}/.env-ng-shared-defs
+[[ -f ${SCBI_ENV_DIR}/.env-shared-defs ]] &&
```

Comment:
No, you need to fully remove this. It is only needed for NG mode for building already provided module.